jiant/preprocess.py

```diff
@@ -552,6 +552,15 @@ def get_tasks(args: config.Params) -> (List[Task], List[str], List[str]):
         # Count examples, store in example_counts.
         if task.example_counts is None:
             task.count_examples()
+        utils.assert_for_log(
+            (
+                task.example_counts["train"] % args.batch_size > len(args.cuda) - 1
```

Comment:
Readability (minor): please use `... >= len(args.cuda)` instead of `... > len(args.cuda) - 1`